src/_pytest/runner.py

```diff
@@ -226,6 +237,8 @@ class CallInfo:
     excinfo = attr.ib(type=Optional[ExceptionInfo])
     start = attr.ib()
```

Comment:
Not sure how to add docstrings to `attrs` classes, but would be nice to add at least types (see `excinfo` example) and code comments on `start`, `stop` and `duration`:

- start (float): The system time when the call started, in seconds since the epoch.
- end (float): The system time when the call ended, in seconds since the epoch..
- duration (float): (Since 5.5.0) The duration of the call, in seconds. Note: this uses an accurate monotonic clock, and should be preferred over using `end - start`.

Reply:
+1 . This goes beyond this precise PR, so I'd rather ask for feedback from other team members so that we do this consistently with the rest. Of course we can also switch to [`pyfields`](https://smarie.github.io/python-pyfields/) which supports having a `doc` argument :) (sorry for the little selfish ad here)

Reply:
See https://github.com/python-attrs/attrs/issues/8#issuecomment-108548968

The suggested way is to include these as `:param <name>: ` in the class docstring since it seems that sphinx supports it, but I do not know if `black` will like it. I push this and we'll see.